src/main/java/bio/terra/service/snapshot/SnapshotService.java

```diff
@@ -338,6 +338,14 @@ private void conjureSnapshotTablesFromAsset(AssetSpecification asset,
         snapshot.snapshotTables(tableList);
     }
 
+    private Column toSnapshotColumn(Column datasetColumn) {
```

Comment:
Is there a reason to not make this a `Column` method? My usual rule of thumb is, if a method accepts an object and operates only on that object (that is, the method could be `static`, as this one could be), then it's likely that it should be a method on that object. Unless its implementation is very specific to a particular class need.